pkg/resource/testdata/TestStatefulSetBuilder/insecure_statefulset_with_resources.golden

```diff
@@ -81,15 +81,34 @@ spec:
         volumeMounts:
         - mountPath: /cockroach/cockroach-data/
           name: datadir
+      securityContext:
+        fsGroup: 10001
+        runAsUser: 10001
       serviceAccountName: cockroach-database-sa
       terminationGracePeriodSeconds: 60
       volumes:
       - name: datadir
         persistentVolumeClaim:
           claimName: ""
         name: datadir
+      initContainers:
+        - command:
+          - /bin/sh
+          - -c
+          - '>- cp -p /cockroach/cockroach-certs-prestage/..data/* /cockroach/cockroach-certs/
+            && chmod 700 /cockroach/cockroach-certs/*.key && chown 10001:10001 /cockroach/cockroach-certs/*.key'
+          image: cockroachdb/cockroach:v19.2.6
```

Comment:
this isn't a versioned test, so this image needs to be updated to v.20.2.x please